cardano-cli/src/Cardano/CLI/Run/Debug/LogEpochState.hs

```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE NamedFieldPuns #-}
+
+module Cardano.CLI.Run.Debug.LogEpochState
+  ( runLogEpochStateCmd
+  ) where
+
+import           Cardano.Api
+import qualified Cardano.Api as Api
+
+import           Cardano.CLI.Commands.Debug.LogEpochState
+import           Cardano.CLI.Orphans ()
+
+import           Data.Aeson
+import qualified Data.ByteString.Lazy.Char8 as BSC
+import qualified System.IO as IO
+
+runLogEpochStateCmd
+  :: LogEpochStateCmdArgs
+  -> IO ()
+runLogEpochStateCmd
+    LogEpochStateCmdArgs
+      { nodeSocketPath
+      , configurationFile
+      , outputFilePath = File outputFilePath
+      } = do
+  appendFile outputFilePath ""
```

Comment:
Why it's needed?

Reply:
It's copied from @Jimbo4350's other PR.  I believe it ensures the file exists.